### ISSUE TITLE
where is train mode of resnet50_coco_best_v2.1.0.h5?

### ISSUE THREAD

**Issue by DenceChen** [fizyr/keras-retinanet#907]
sorry I only download the resnet50_coco_best_v2.1.0.h5 , this mode is a inference model, but i want a trian model of resnet50_coco_best_v2.1.0.h5 for finetune, not just use **--weight**, can you show me the link? ths all

**Comment by hgaiser** [comment]
You can use the inference model for fine-tuning, it contains all the weights of the training model plus some additional layers to output bounding boxes.
